__tests__/app.test.js

```diff
@@ -45,8 +45,108 @@ describe("GET /api/topics", () => {
             slug: "paper",
           },
         ];
-        expect(body.topics).toMatchObject(expectedResponse);
+        expect(body.topics).toEqual(expectedResponse);
         expect(body.topics.length).toBe(3);
       });
   });
 });
+
+describe("GET /api/blah", () => {
+  it("responds with status 400 for bad path message", () => {
+    return request(app)
+      .get("/api/blah")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("URL not found");
+      });
+  });
+});

```

Comment:
Is this supposed to be 400 or 404?

Reply:
I believe this is 404 as it is wrong path for api call